### ISSUE TITLE
Do not hide scanner results on review tools page

### ISSUE THREAD

**Issue by wagnerand** [mozilla/addons-server#20233]
Under some circumstances, we do not show the scanner result for a version on the review page anymore. I think at least one scenario is when the version has been reviewed by a human. There might be other cases.

We should always show the scanner result on the review page, except if that result has been marked as a false positive.

**Comment by diox** [comment]
Scanner results are currently shown as long as the version has been flagged by a scanner action as needing human review. Nothing has changed recently about that. 

The only recent change is that https://github.com/mozilla/addons-server/issues/19825 made that information available to all reviewers instead of requiring the user to be staff + `ADMIN_SCANNERS_RESULTS_VIEW` (and it also added a lot more detail to what we display)

**Comment by wagnerand** [comment]
Yes, I don't believe this is a new issue. But with more folks having access to this information, this would be more helpful now than before.

**Comment by ioanarusiczki** [comment]
@diox @wagnerand 

I've uploaded and marked as false positive results for[ this addon ](https://reviewers.addons-dev.allizom.org/en-US/reviewers/review/624843). An addon reviewer can see the scanner results. After them being marked false positive they're still visible on the reviewer page.

I also tried for an [unlisted addon](https://reviewers.addons-dev.allizom.org/en-US/reviewers/review-unlisted/624847) An addon reviewer with Addons:ReviewUnlisted permissions can see the results on the review page. I approved the version and marked the results false positive -> they are still displayed on the review-unlisted page.
